src/Annotations/AppKernel.php

```diff
@@ -224,7 +224,10 @@ public function __construct(string $environment = 'dev')
 
             return new Client([
                 'base_uri' => $this->app['hypothesis']['api_url'],
+                'connect_timeout' => 0.5,
+                'decode_content' => 'gzip',
                 'handler' => $this->app['hypothesis.guzzle.handler'],
+                'timeout' => 0.9,
```

Comment:
We want fast failures as Journal has a 1-second timeout. This has to be less than that otherwise Annotations won't see an error (Journal will just have disconnected).

Reply:
It's reasonable to think that the rest of the code being executed by this service will have a 100ms overhead